pkg/resourcecache/main.go

```diff
@@ -15,6 +15,7 @@ type ResourceCache interface {
 	CreateResourceInformer(resource string) (GenericCache, error)
 	StopResourceInformer(resource string)
 	GetGVRCache(resource string) (GenericCache, bool)
+	CreateResourceInformerByGVK(apiVersion string, kind string) (GenericCache, error)
```

Comment:
Can we remove `CreateResourceInformer` and use`CreateResourceInformerByGVK` instead? 

The only difference is how we fetch the `gvr`: `apiResource, gvr, err := resc.dclient.DiscoveryClient.FindResource(apiVersion, kind)`

Reply:
okay.